bundles/af-core/src/test/resources/form/textinput/exporter-multiline-textinput.json

```diff
@@ -5,11 +5,9 @@
   "visible": false,
   "description": "dummy",
   "type": "string",
-  "required": false,
   "enabled": true,
   "readOnly": false,
   "label": {
-    "richText": false,
     "visible": true,
```

Comment:
visible should be removed.

Reply:
not sure why should we remove it, we have removed the default values from sling model , but if the dialog is used to set a value we should still test the sling model behaviour right?

Reply:
resolved as per discussion